src/Validation/Validation.php

```diff
@@ -876,6 +884,10 @@ public static function maxLength($check, $max)
      */
     public static function minLengthBytes($check, $min)
     {
+        if (!is_scalar($check)) {
```

Comment:
Couldn't a resource be checked against its byte length?

Reply:
How would a resource come in request data?